src/common/mfu_io.c

```diff
@@ -609,6 +669,8 @@ void daos_open(const char* file, int flags, mode_t mode, mfu_file_t* mfu_file)
 
     mfu_free(&name);
     mfu_free(&dir_name);
+
+    return -1;
```

Comment:
Should this be ``return rc``?

Also, if ``#ifdef DAOS_SUPPORT`` is not active, I'm guessing some compilers will not be happy with this function, since it is declared to return an int, but it wouldn't have a return statement in that case.  To fix that, you could add an ``#else`` case or perhaps but an ``int rc = 0;`` at the top and an ``return rc;`` at the bottom?

Reply:
On second thought, I see the ``#ifdef`` is the same on all daos functions, so let's wait to do that in a future PR if it's actually needed.

Reply:
> Should this be `return 0`?
Yes, that was careless of me... I'll need to investigate my tests didn't catch this...
But it has been fixed.

Reply:
> `int rc = 0;` at the top and an `return rc;` at the bottom

I think this should be easy enough to do. I can do that when I also go through this one:
https://github.com/hpc/mpifileutils/issues/382
Since I will be touching most functions anyway.